gcam-usa.md

```diff
@@ -216,6 +216,32 @@ Non-CO2 GHGs
+GCAM-USA includes state-level emissions from the following air pollutants: BC, OC, PM2.5, PM10, NOx, SO2, NMVOC, CO, and NH3 for the following aggregate sectors: electric generation, buildings, onroad transportation, nonroad transportation, industrial energy use, industry processes, urban processes, cement, and refining and related. Depending on the emissions and energy data available, either emission factors (EFs) or input emissions are used. These EFs or emissions are calibrated in all historical years to the US [National Emissions Inventory](gcam-usa.html#nei) (NEI), allowing for state-level representations of pollutant emissions. Annual data at the state level from [EPA Tier 1 CAPS](gcam-usa.html#epatier1) was used to scale some of the NEI data to get a more accurate distribution of emissions for years in between, or before, NEI years. National totals for historical years are scaled to [Community Emissions Data System](gcam-usa.html#ceds) (CEDS) values for most sectors at the national level both for consistency over time and with the global GCAM model, and because CEDS values are updated annually, providing up to date estimates. Future emission factors are set, by vintage where available in GCAM, according to current air pollution control regulations. Outlier emission factors are reset to national median values to avoid spurious results. In some sectors, supplemental data from [MOVES](gcam-usa.html#moves), [MARKAL](gcam-usa.html#markal), [GREET](gcam-usa.html#greet), [IMO](gcam-usa.html#imo), and various other EPA sources.
+
+#### Additional Information and Assumptions
+BC and OC: The NEI does not include BC/OC emissions. These are derived from PM2.5 emissions using BC/OC ratios. There can be some inconsistency between GCAM USA BC/OC emissions and PM2.5 emissions. In practice this is usually not a significant issue because most analysis uses either BC/OC (climate-focused analysis) or PM2.5 (air pollution focused analysis) emissions, but not both.
+
+For electric generation, only the GCAM-USA configuration with water technologies is supported.
+
+For industrial energy use, emission factors for future years are applied assuming industrial sector vintaging is being used, which could lead to discontinuity if not.
+
+Note that the air pollutant representation in GCAM-USA takes a different paradigm than in core GCAM. Air pollutant emissions are implemented as a CLE (Current LEgislation) scenario, reflecting current rules and legislation with one additional GDP control in place, although the impact of any endogenous technology changes are taken into account. This is done to support work on energy-emissions interactions over the next 20-30 years (out to 2050). Some emissions start to increase again by that point. This behavior is especially seen in nonroad transportation due to minimal legislation represented that impacts future year emissions as well as a lack of GDP control. Table 2 reflects the CLE approach by sector.

```

Comment:
- I suggest removing "Note that", and simply starting with "The air pollutant representation ...".
- "GCAM-USA takes a different paradigm":  I suggest "employs" or utilizes" rather than "takes"
- "Air pollutant emissions are implemented as a CLE ...":  I think we should be more specific, e.g., "Future air pollutant emissions factors are implemented as a CLE ..."
- "with one additional GDP control in place, although the impact of any endogenous technology changes are taken into account."  This is a little too vague for me.  The GDP control is only applied to one sector, right?  How is the impact of endogenous technology changes taken into account.  I recommend ending the sentence after "reflecting current rules and legislation", and then adding additional sentences with more detail on the GDP control and endogenous technology change.
- "This is done to support work on energy-emissions interactions over the next 20-30 years (out to 2050)."  I suggest "This implementation is focused on energy-emissions interactions over the next 20-30 years (out to 2050); some emissions begin to increase again after 2050 as the underlying activity drivers increase and emissions factors remain constant (i.e., do not reflect further emission controls)."
- "Table 2 reflects...":  I suggest "outlines" instead of "reflects".